src/common/common.ts

```diff
@@ -0,0 +1,27 @@
+import { toHexString } from './hex-strings'
+
+export const assert = (condition: () => boolean, reason?: string) => {
+  try {
+    if (condition() === false) {
+      throw new Error(`Assertion failed: ${reason}`)
+    }
+  } catch (err) {
+    throw new Error(`Assertion failed: ${reason}\n${err}`)
+  }
+}
+
+export const toRpcHexString = (n: number): string => {
```

Comment:
@platocrat we could reopen this issue and get it merged if we only include the contents of this file.

Reply:
Actually, even better would be to move `toRpcHexString` and `padHexString` to `common/hex-strings.ts`.

Reply:
I'll reopen it rn